manifests/07_deployment.yaml

```diff
@@ -35,7 +35,8 @@ spec:
               echo "Copying system trust bundle"
               cp -f /var/run/configmaps/trusted-ca-bundle/ca-bundle.crt /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
           fi
-          exec authentication-operator operator --config=/var/run/configmaps/config/operator-config.yaml --v=2 --terminate-on-files=/var/run/configmaps/trusted-ca-bundle/ca-bundle.crt
+          touch /tmp/terminate
```

Comment:
The observer should be capable of watching even just a creation of the file, we shouldn't need to create it.

The termination would then happen simply by performing some sort of "touch" from Go.

Reply:
done